components/CheckStatusResponses/CheckStatusShippingFedex.tsx

```diff
@@ -12,20 +12,21 @@ export const CheckStatusShippingFedex: FC<CheckStatusShippingFedexProps> = ({
   const { t } = useTranslation(['status', 'common'])
   return (
     <>
-      <p data-testid="shipped-fedex">
-        {t('shipped-fedex.printed-and-mailing')}
-      </p>
+      <h2 data-testid="shipped-fedex" className="h2 text-blue-normal">
+        {t('status-check-passport-printed')}
+      </h2>
+      <span>{t('shipped-fedex.mailing')}</span>
```

Comment:
Should be wrapped with a `p` tag.

```tsx
<p>
        {t('shipped-fedex.mailing')}
        {trackingNumber && (
          <>
            {t('status-check-tracking.number')} <b>{trackingNumber}</b>.<br />
            <ExternalLink
              href={t('status-check-tracking.link.fedex', { trackingNumber })}
            >
              {t('status-check-tracking.link.text')}
            </ExternalLink>
          </>
        )}
      </p>
```

`\n` shouldn't be used for the link text